packages/apps/hr/src/components/templates/ExpensePicker/ExpensePicker.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useEffect} from 'react';
+import {StyleSheet} from 'react-native';
+import {Picker} from '@axelor/aos-mobile-ui';
+import {useSelector, useDispatch} from '@axelor/aos-mobile-core';
+import {searchExpenseDraft} from '../../../features/expenseSlice';
+
+const ExpensePickerAux = ({
```

Comment:
Don't we already have a picker with user's draft expenses ? On the popup to link orphan lines to an expense